2020/questions.md

```diff
@@ -100,8 +102,10 @@ NEW QUESTION
 * IT
 * Office of the CTO
 * Developer relations, marketing or communications
+* Security and risk compliance
```

Comment:
```suggestion
* Security, compliance, risk management
```

"risk compliance" feels off

Reply:
You're right. My bad. Last year, in "Other", "Security and Risk Management" and "Information Security/GRC"